### ISSUE TITLE
totalRevenues values are missing from NVDA 2019 10-K

### ISSUE THREAD

**Issue by wongni** [john-friedman/datamule-python#97]
Filling URL: https://www.sec.gov/Archives/edgar/data/0001045810/000104581019000023/

<img width="925" height="264" alt="Image" src="https://github.com/user-attachments/assets/ddc84632-7032-4cdb-9409-fdd633347028" />

submission.fundamentals["incomeStatement"] does not have "totalRevenues"

It has the following metrics. 

<img width="223" height="365" alt="Image" src="https://github.com/user-attachments/assets/273f2405-7c38-40e8-b90e-d5f519bdc853" />
